providers/fips/fipsprov.c

```diff
@@ -316,6 +345,8 @@ int OSSL_provider_init(const OSSL_PROVIDER *provider,
     FIPS_GLOBAL *fgbl;
     OPENSSL_CTX *ctx;
 
+    memset(&selftest_params, 0, sizeof(selftest_params));
```

Comment:
This assumes that the FIPS provider will never be initialised more than once concurrently. That may not be true. I'm not sure why its necessary anyway.

Reply:
true its static.. so removed..